### PR TITLE
[AIRFLOW-4135] Google Cloud Build

### PR DESCRIPTION
**Pull request by mik-laj**

### CODE REVIEW

File: airflow/contrib/operators/gcp_cloud_build_operator.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    :param body: The request body.
+        See: https://cloud.google.com/cloud-build/docs/api/reference/rest/Shared.Types/Build
+    :type body: dict
+    :param gcp_conn_id: The connection ID to use to connect to Google Cloud Platform.
+    :type gcp_conn_id: str
+    :param api_version: API version used (for example v1 or v1beta1).
+    :type api_version: str
+    """
+
+    # [START gcf_function_delete_template_fields]
```
Comment (antonimaciej) [inline airflow/contrib/operators/gcp_cloud_build_operator.py]: I think this flag's name is a copy-paste leftower

File: docs/conf.py [hunk 2]
```diff
@@ -160,7 +160,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = ['_build', 'howto/operator/gcp/_partials']
```
Comment (antonimaciej) [inline docs/conf.py]: Didn't you try to sneak in this "partials" change with some other commit? ;>

Comment (mik-laj) [reply docs/conf.py]: Yes. I do it. This file is already on apache/master

File: airflow/contrib/operators/gcp_cloud_build_operator.py (hunk 1 above)
Comment (mik-laj) [reply airflow/contrib/operators/gcp_cloud_build_operator.py]: I updated a PR. Currently, these markers are not needed in the code.

File: docs/conf.py (hunk 2 above)
Comment (mik-laj) [reply docs/conf.py]: I updated a PR.

**Pull request closed without merging**